packages/web-react/src/components/Toast/demo/ToastDynamicToastQueue.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { ChangeEvent, useEffect, useState } from 'react';
+import { AlignmentXDictionaryType, AlignmentYDictionaryType, ToastColorType } from '../../../types';
+import { Button } from '../../Button';
+import { Checkbox } from '../../Checkbox';
+import { Stack } from '../../Stack';
+import { Select } from '../../Select';
+import { TextArea } from '../../TextArea';
+import { Radio } from '../../Radio';
+import { Link } from '../../Link';
+import { TextField } from '../../TextField';
+import Toast from '../Toast';
+import ToastBar from '../ToastBar';
+import { useToast } from '../useToast';
+import { ToastItem } from '../ToastContext';
+
+const ToastDynamicToastQueue = () => {
+  const [isCollapsible, setIsCollapsible] = useState(true);
+  const [alignmentY, setAlignmentY] = useState<AlignmentYDictionaryType>('bottom');
+  const [alignmentX, setAlignmentX] = useState<AlignmentXDictionaryType>('center');
+  const [colorValue, setColorValue] = useState<ToastColorType>('inverted');
+  const [hasIconValue, setHasIconValue] = useState(true);
+  const [isDismissibleValue, setIsDismissibleValue] = useState(true);
+  const [messageValue, setMessageValue] = useState('This is a new toast message.');
+
+  const { queue, show, hide, clear, setQueue } = useToast();
+
+  const defaultToastQueue: ToastItem[] = [
+    {
+      id: '1',
+      isOpen: true,
+      message: (
+        <>
+          I was first!{' '}
+          <Link href="#" color="inverted" isUnderlined>
+            Action
+          </Link>
+        </>
+      ),
+      color: 'success',
+      hasIcon: true,
+      isDismissible: true,
+      iconName: undefined,
+    },
+    {
+      id: '2',
+      message: (
+        <>
+          I appeared later. This toast has a long message that wraps automatically.{' '}
+          <Link href="#" color="inverted" isUnderlined>
+            Action
+          </Link>
+        </>
+      ),
+      isOpen: true,
+      color: 'informative',
+      hasIcon: true,
+      isDismissible: true,
+      iconName: undefined,
+    },
+  ];
+
+  useEffect(() => {
+    setQueue(defaultToastQueue);
+  }, []);
+
+  const handleAlignmentYChange = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleAlignmentYChange = (event: ChangeEvent<HTMLInputElement>) => {
```

Can we use a full name, please?

Reply:
renamed